cert/cert.go

```diff
@@ -331,7 +330,7 @@ func (spec *Spec) Identity() (*core.Identity, error) {
 	return ident, nil
 }
 
-func readCertFile(path string) (*Spec, error) {
+func NewSpecFromPath(path string) (*Spec, error) {
```

Comment:
Exported functions should have a comment above for godoc (linter will complain).

Reply:
Oddly the travis linter didn't complain about this, but agreed.

Longer term, this function is going to go into a config.go file (along w/ the equivalent manager functions).

Reply:
making this private until I move it into a config.go